.github/ISSUE_TEMPLATE/02_questions.md

```diff
@@ -1,17 +1,10 @@
 ---
 name: "\U00002753 Questions?"
-about: If you have *specific* questions about the website, please post them here.
+about: This issue template should not be used anymore!
 labels: question
 
 ---
-<!--
-Thanks for submitting your question 🙌 ❤️
 
-Before opening a new issue, please make sure that we do not have any duplicates already open. You can ensure this by searching the issue list for this repository. If there is a duplicate, please close your issue and add a comment to the existing issue instead. Also, please, have a look at our FAQs and existing questions before opening a new question.
--->
+## Don't open issues for questions!
 
-## Your Question
-<!-- Include details about your question. -->
-* Website URL Path:
-* Line / Paragraph:
-* Question:
+Please **don't** open new issues for questions anymore. Instead, head to the [discussion's section](https://github.com/corona-warn-app/cwa-website/discussions) and create a new discussion choosing the Category "Q&A".
```

Comment:
I suggest to align with the GitHub help https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#creating-a-discussion and write

`click on  [Discussions](https://github.com/corona-warn-app/cwa-website/discussions)`instead of

`head to the [discussion's section](https://github.com/corona-warn-app/cwa-website/discussions)`